03/app.js

```diff
@@ -1,2 +1,10 @@
 const n = 24;
-const oddNumbers = [];
\ No newline at end of file
+const oddNumbers = [];
+
+for (let i = 1; i <= n; i++) {
+  if (i % 2 !== 0) {
```

Comment:
Można też `i % 2 === 1` - jest czytelniej moim zdaniem. Zaprzeczenie to dodatkowa rzecz do "przetrawienia" :D